grpc/src/byte_str.rs

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
probably before we merge we should update the license in accordance to https://github.com/cncf/foundation/issues/985#issuecomment-2949404196